drop-transfer/src/service.rs

```diff
@@ -85,14 +89,39 @@ impl Service {
 
             ws::server::spawn(state.clone(), logger.clone(), stop.clone(), guard.clone())?;
 
-            manager::resume(&state, &logger, &guard, &stop).await;
-
-            Ok(Self {
+            let mut service = Self {
                 state,
                 stop,
                 waiter,
                 logger,
-            })
+                tasks: Default::default(),
+            };
+
+            let outgoing_transfers = {
+                let xfers = service.state.transfer_manager.outgoing.lock().await;
+                xfers
+                    .values()
+                    .map(|xstate| xstate.xfer.clone())
+                    .collect::<Vec<_>>()
+            };
+
+            for xfer in outgoing_transfers {
+                service.trigger_peer_outgoing(xfer);
+            }
+
+            let incoming_transfers = {
+                let xfers = service.state.transfer_manager.incoming.lock().await;
+                xfers
+                    .values()
+                    .map(|xstate| xstate.xfer.clone())
+                    .collect::<Vec<_>>()
+            };
+
+            for xfer in incoming_transfers {
+                service.trigger_peer_incoming(xfer);
+            }
```

Comment:
I thought that we agreed to remove the triggering of of transfers on service start

Reply:
No idea how it slipped back in(or possibly was never removed). Pushed the change just now